var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -74,6 +74,7 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
         "hwloc": [False, "Whether to enable the HWLOC library"],
         "numactl": [False, "Whether to enable the LIBNUMA library"],
         "memkind": [False, "Whether to enable the MEMKIND library"],
+        "rocthrust": [False, "Whether to enable the thrust library for the HIP backend"],
```

Comment:
It is a bit more complicated: `rocthrust` should be enabled by default when `+rocm` and disable else.